FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Extensions/ConfigurationRegistrationExtensions.cs

```diff
@@ -16,29 +19,21 @@ public static class ConfigurationRegistrationExtensions
             this IServiceCollection services,
             IConfiguration configuration)
         {
```

Comment:
Need to check configVersion is null here?

Reply:
Null will aslo be resolved to other versions when in `TryParse`